collect_app/src/main/java/org/odk/collect/android/savepoints/SavepointFinder.kt

```diff
@@ -0,0 +1,48 @@
+package org.odk.collect.android.savepoints
+
+import android.net.Uri
+import org.odk.collect.android.external.FormsContract
+import org.odk.collect.android.utilities.ContentUriHelper
+import org.odk.collect.forms.FormsRepository
+import org.odk.collect.forms.instances.InstancesRepository
+import org.odk.collect.forms.savepoints.Savepoint
+import org.odk.collect.forms.savepoints.SavepointsRepository
+import java.io.File
+
+object SavepointFinder {
+    fun getSavepoint(
```

Comment:
This could maybe live on `FormEntryUseCases` instead of a new object? Alternatively, we could call this object `SavepointUseCases` if it feels separate to you.

Reply:
I like SavepointUseCases more.